### ISSUE TITLE
Error on Build: 'EditorCoroutines' does not exist in the namespace 'Unity' 

### ISSUE THREAD

**Issue by EricBatlle** [ZeShmoutt/Blizzard-API-For-Unity#5]
When trying to compile the project, it shows the following error like 30 times.: 

`Library\PackageCache\com.zeshmouttsassets.blizzardapi@1370fe6d99\Scripts\_Core\Editor\BlizzardAPI_AccessTokenWindow.cs(2,13): error CS0234: The type or namespace name 'EditorCoroutines' does not exist in the namespace 'Unity' (are you missing an assembly reference?)`

I'm using Unity 2019.3.13f1, following manual installation, targeting Android and PC on an empty clean project, that only have EditorCoroutine package and Blizzard-API-For-Unity package.

How can I solve it? Thanks

**Comment by ZeShmoutt** [comment]
Can't reproduce it.

You installed it with the package manager this time, right ?

**Comment by ZeShmoutt** [comment]
That makes no sense, editor scripts should be ignored when building.

I'll investigate.

You installed it with the package manager this time, right ?
![image](https://user-images.githubusercontent.com/36408045/95139276-635f8980-076c-11eb-9941-e3ddfe2c4568.png)

**Comment by EricBatlle** [comment]
> You installed it with the package manager this time, right ?
> ![image](https://user-images.githubusercontent.com/36408045/95139276-635f8980-076c-11eb-9941-e3ddfe2c4568.png)

Yes, have you been able to reproduce the  bug? or it's happening only to me?

**Comment by ZeShmoutt** [comment]
I could reproduce it, but as I said it makes no sense. All 31 errors are from editor-related namespaces/classes in an Editor folder, which should be completely ignored for builds.

... or at least that's what I thought, until I found [this thread](https://forum.unity.com/threads/player-build-error-with-editor-folders.556864/). Turns out assembly definition files (which are required to make a proper package) do *not* ignore Editor folders.

Doesn't look too hard to fix. I'll try to submit something after work.
